distribution/src/config/elasticsearch.yml

```diff
@@ -80,3 +80,16 @@
 # Require explicit names when deleting indices:
 #
 #action.destructive_requires_name: true
+#
+# ---------------------------------- Security ----------------------------------
+#
+#                                 *** WARNING ***
+#
+# Elasticsearch security features are disabled by default.
```

Comment:
This isn't true for paid (Gold+ licenses) and I wonder whether the message might scare some people.
I don't have a good suggestion, but if we can come up with something that still makes a strong statement with a little more accuracy.

Maybe

```suggestion
# Elasticsearch security features are not enabled by default.
# These features are free, but require configuration changes to enable them.
```

Reply:
I'm 👍 to this change. I think that brevity wins here.